.travis.yml

```diff
@@ -3,7 +3,7 @@ before_install:
 language: rust
 rust:
   # If the MSRV is updated, update the readme
-  - 1.65.0
+  - 1.81.0
```

Comment:
are you guys thinking of opening a benchmark CI, with that external portal that I can't remember the name ?